_includes/desktop-table.html

```diff
@@ -16,12 +16,19 @@
     </thead>
     <tbody class="jets-content">
     {% for website in section_file.websites %}
+    {% assign email_subject  = page.email_subject  %}
+    {% assign progress_tweet = page.progress_tweet %}
+    {% assign workonit_tweet = page.workonit_tweet %}
```

Comment:
I'd go for using `site.data.language["en"].` for this three lines (and for the same in the mobile-table.html), because that would eliminate the need to have the same variables added in index.html and thus remove the need to change them at two different places.
And because we need them in the `languages`-data we could strip them from index.html.
// Kai